webapp/channels/src/components/advanced_text_editor/advanced_text_editor.test.tsx

```diff
@@ -0,0 +1,326 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {screen} from '@testing-library/react';
+import React from 'react';
+
+import type {Channel} from '@mattermost/types/channels';
+
+import Permissions from 'mattermost-redux/constants/permissions';
+
+import type {FileUpload} from 'components/file_upload/file_upload';
+import type Textbox from 'components/textbox/textbox';
+
+import WebSocketClient from 'client/web_websocket_client';
+import mergeObjects from 'packages/mattermost-redux/test/merge_objects';
+import {renderWithRealStore, userEvent} from 'tests/react_testing_utils';
+import {WebSocketContext} from 'utils/use_websocket';
+
+import type {PostDraft} from 'types/store/draft';
+
+import AdavancedTextEditor from './advanced_text_editor';
+
+global.ResizeObserver = require('resize-observer-polyfill');
+
+const currentUserId = 'current_user_id';
+const channelId = 'current_channel_id';
+
+const initialState = {
```

Comment:
Is all of this needed when using `renderWithRealStore`/`renderWithFullContext`? Those should be initializing the empty store state like the web app initially has so you shouldn't need to initialize empty fields like `entities.groups.groups` and such

Reply:
For some reason I thought I needed all these, but removing them still make the tests pass, so... I guess we are fine not having them :smile: